nursery/get-system-information-in-dotnet-on-android.yml

```diff
@@ -0,0 +1,10 @@
+rule:
+  meta:
+    name: get system information in .NET on Android
```

Comment:
seems rather generic. could we make this name a little more specific? is it the OS version or similar?